src/asset-swapper/quote_consumers/exchange_proxy_swap_quote_consumer.ts

```diff
@@ -492,4 +504,33 @@ export class ExchangeProxySwapQuoteConsumer implements SwapQuoteConsumer {
                 .getABIEncodedTransactionData();
         }
     }
+
+    private encodeUniswapV2FillCalldata(quote: SwapQuote, opts: ExchangeProxyContractOpts & { takerAddress?: string }): string {
```

Comment:
I saw that you created a method for swapExactTokensForTokensSupportingFeeOnTransferTokens , what about for:
- swapExactETHForTokensSupportingFeeOnTransferTokens  --> FUSE (native) to fee token
- swapExactTokensForETHSupportingFeeOnTransferTokens  --> Fee token to FUSE (native)

Do we need to add support for those cases?

Reply:
@mul53 ?

Reply:
good catch @darkpaladi, the PR now should be okay